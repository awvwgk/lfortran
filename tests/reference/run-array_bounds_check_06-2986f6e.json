{
    "basename": "run-array_bounds_check_06-2986f6e",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_06.f90",
    "infile_hash": "8d6d28fe1c81ff7cba66afc6e853bd26e5b8716b40fd41942df789a8",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_06-2986f6e.stderr",
    "stderr_hash": "efe0198d97293d4c98ab05eda3f3fb4a6cc6bcdd3bd6e93a3402c16d",
    "returncode": 1
}