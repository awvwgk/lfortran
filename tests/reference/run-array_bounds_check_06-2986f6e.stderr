At 10:5 of file tests/errors/array_bounds_check_06.f90
Runtime Error: Size mismatch in assignment to 'x'

LHS size is 3 and RHS size is 4
